client/app/components/Input/__tests__/InputRadio.spec.jsx

```diff
@@ -0,0 +1,62 @@
+// @flow
+import React from 'react';
+import { shallow } from 'enzyme';
+import { act } from 'react-dom/test-utils';
+import { InputRadio } from '../InputRadio';
+import { InputMocks } from '../../../mocks/InputMocks';
+
+const {
+  id, name, value, options,
+} = InputMocks.inputRadioProps;
+const someEvent = InputMocks.event;
+
+describe('InputRadio', () => {
+  beforeEach(() => {
+    jest.spyOn(window, 'alert');
+  });
+
+  it('sets default radio button to first option', () => {
+    const wrapper = shallow(
+      <InputRadio
+        name={name}
+        id={id}
+        value={value}
+        options={options}
+        onChange={someEvent}
+      />,
+    );
+    expect(
+      wrapper.find('input[id="some-option-one-id"][type="radio"]').props().checked,
+    ).toBeTruthy();
```

Comment:
I think you can be more explicit here and use the `.toBeTrue()` assertion since we expect booleans. Same case below too.

Reply:
Getting an error that `.toBeTrue()` is not a function using the current `jest-jasmine` testing library 🤔 but I did change this to `.toBe(true)` if that also is explicit in this case?

Reply:
Oh that works too! :)